frontend/src/App.tsx

```diff
@@ -217,14 +221,40 @@ function App(props: { setOnDisconnect: Dispatch<SetStateAction<Callback | undefi
   }, [dispatchAppUpdate, setOnDisconnect]);
 
   const page = useMemo(() => {
-    if (!appState.sessionToken) {
-      return <Login doLogin={setupGameController} />;
-    } if (!videoInstance) {
-      return <div>Loading...</div>;
-    }
+    // TODO: This is only temporary. This needs to be removed
+    console.log("DEBUG: REMOVE ME PLEASE!")
+    // if (!appState.sessionToken) {
+    //   return <Login doLogin={setupGameController} />;
+    // } if (!videoInstance) {
+    //   return <div>Loading...</div>;
+    // }
+
     return (
-      <div>
-        <WorldMap />
+      <div style={{
+        display: "flex",
+        flexDirection: "column",
+      }}>
+        {/* 
+        TODO: This needs to be cleaned up. Phaser is not respecting CSS rules apparently.
```

Comment:
So for whatever reason, the phaser component doesn't really care about CSS positioning. I'll tackle this problem in another PR, since it was taking up a lot of time. Keeping this TODO intact, and will remove this later on.